serverless/containers/reference-content/containers-limitations.mdx

```diff
@@ -26,6 +26,8 @@ This section contains usage limits that apply when using Serverless Containers.
 | Invocation rate                                   | Max Number  | 1000 per second  | Container                   |
 | Concurrency                                       | Max         | 80               | Container Instance          |
 | Scale (simultaneous Container Instances)          | Max         | 20               | Container                   |
+| CPU limit                                         | Max         | 2240 mvCPU       | Container                   |
```

Comment:
Maybe in one line it makes more sense ? For example:

`| CPU                                         | Min/Max         | 70 to 2240 mvCPU       | Container                   |`